upgrade/helpers/constants/__init__.py

```diff
@@ -0,0 +1,10 @@
+from dynaconf import Dynaconf
```

Comment:
I think this should not be in constants at all ! 

DynaConf itself suggests its a Dynamic settings, So I would suggest to create `upgrade/config/__init__.py` instead and put these there.

You can also set nailgun from this file for all to use.

Reply:
Refer robottelo `config/__init__.py` instead.